app/helpers/physical_server_helper/textual_summary.rb

```diff
@@ -1,10 +1,16 @@
 module PhysicalServerHelper::TextualSummary
   def textual_group_properties
-    %i(name model product_name manufacturer machine_type serial_number ems_ref)
+    TextualGroup.new(
+      _("Properties"),
+      %i(name model product_name manufacturer machine_type serial_number ems_ref)
+    )
   end
 
   def textual_group_relationships
-    %i(host)
+    TextualGroup.new(
+      _("Relationships"),
+      %i(host)
```

Comment:
As below, you see the 'mlk' summary with its relationship with 'Imm 0901' host:
![captura de tela de 2017-05-09 13-04-50](https://cloud.githubusercontent.com/assets/17187082/25860346/a613cb1e-34b7-11e7-9e1f-eb64784d6ef1.png)

After I click in host, it loads this specific host page.

![captura de tela de 2017-05-09 13-05-16](https://cloud.githubusercontent.com/assets/17187082/25860345/a6136476-34b7-11e7-933e-e0f09c0a22af.png)

Btw the Physical Infrastructure relationship shown will be added in another Pull Request after https://github.com/ManageIQ/manageiq-ui-classic/pull/1294 is merged.